_config.yml

```diff
@@ -33,6 +33,8 @@ assets:
   sources:
     - _assets/bower
     - _assets/fonts
+    - _assets/fonts/Cormorant-Garamond
```

Comment:
I would have thought. It will not work without including them, test on my macOS and Debian 8.
These child folders in **fonts folder** will not be loaded in **"_site/assets/"** ..